superset-frontend/src/components/ListView/utils.ts

```diff
@@ -78,6 +82,23 @@ export function convertFilters(fts: InternalFilter[]): FilterValue[] {
     .map(({ value, operator, id }) => ({ value, operator, id }));
 }
 
+// convertFilters but to handle new decoded rison format
+export function convertFiltersRison(filterObj: any): FilterValue[] {
```

Comment:
I noticed before that all unselected filters muddy the urls, can we filter out where `value === undefined`?

Also did we consider going with a better/simpler data structure? maybe 
```js
{
   user: ['==', 'mistercrunch'],
   dataset: ['==', 'some_dataset'],
}
```
`(dataset:!(==,some_dataset),user:!(==,mistercrunch))`

Reply:
I think the operators names were long too, and using things like `==` `<>` `||`  would make for nicer/shorter URLs

Reply:
yes, this update should only put non-default filters in the URL for a much cleaner experience. I'll add some screenshots that show the URL params next to the selected filters. Also want to remove the operators but I'm working through a weird bug around that atm.